src/odemis/driver/xt_client.py

```diff
@@ -2147,10 +2166,17 @@ class Stage(model.Actuator):
     moving the TFS stage and updating the position.
     """
 
-    def __init__(self, name: str, role: str, parent: model.HwComponent, rng: dict = None, **kwargs) -> None:
+    def __init__(self, name: str, role: str, parent: model.HwComponent, rng: dict = None, raw: bool = False,
```

Comment:
"raw" a little bit generic word. I think that when encountering it in a microscope file, a lot of people wouldn't understand what it refers too. You can of course count on comments, but I'd suggest to name it `raw_coordinates` to make it clearer immediately.